tips/TIP-0038/tip-0038.md

```diff
@@ -0,0 +1,1064 @@
+13+3 is true.
+
+**Backdating blocks**
+
+One variable that was not mentioned yet is how a client can set a block's `Issuing Time`, relative to which the chosen
+commitment is validated, which allows for _backdating_ blocks. Backdating blocks by a singificant amount is only
+possible when there is liveness. The most that someone can backdate a block is to
+`Accepted Tangle Time - Liveness Threshold Upper Bound`, so if there is liveness (`ATT ~= Wall Clock Time`), then the
+most someone could backdate their block is by `Liveness Threshold Upper Bound`.
+
+![](./assets/backdating-blocks.png)
+
+_This figure shows an example of backdating a block by one slot (which is less than `Liveness Threshold Upper Bound`).
+The global Wall Clock Time is the same for both blocks and yet both transactions in their respective blocks could unlock
+the output._
+
+The only time that backdating has any consequence is if there is liveness. Assume some application (for example, a layer
+2 chain) relies on an expired output being consumed. The application should only consider the expired output as consumed
+and act on that if the output is at least accepted, but ideally until it is committed, confirmed or finalized. The
+higher the level of finality the application waits for, the more difficult it is for the `Address` owner (the receiver
+of the output with an _Expiration Unlock Condition_) to backdate a block and revert the consumption of the expired
+output. For example, if the application waits for commitment of the expired output being consumed, then it is impossible
+for the receiver address to subsequently consume it, unless it forces a chain switch, because the receiver address can
+only consume the output in an earlier slot than the return address can consume it, and all such earlier slots are
+already committed. If the application only waits for acceptance of the expired output consumption, a malicious owner of
+the receiver address could technically issue a backdated block shortly after `ATT - Liveness Threshold Upper Bound`, but
+this block would be less likely to be selected as a tip, and because the majority of online validators already accepted
+the expired output consumption, they should not accept the backdated receiver consumption because it is a conflict. The
+only way the receiver address could revert the other conflict is by having support of a majority of other validators to
+revert it. In conclusion, it is very difficult to backdate even with just acceptance, and becomes more difficult when
+waiting for higher levels of finality. This attack relates to the fundamental fact that the best approximation of time
+is the block's `Issuing Time` timestamp, which is is meant to be very difficult to falsify.
+
+**Note**: An IOTA Smart Contract Chain is not susceptible to such a backdating attack anyway because it is the receiver
+address of the expiration locked output and it is _not_ waiting for the output to expire in order to consume it.
```

Comment:
But ISC could also create outputs with an expiration unlock condition?

Reply:
Do they actually do that?

And even if they did, when they are the `Return Address` and wait for the expired output to be committed (for example), they have a reasonably high guarantee that they can consume it without the `Address` owner backdating a block and consuming it.

Reply:
Currently not that I'm aware of, but it could be possible
Just don't understand why there is an extra note for this

Reply:
We discussed it in the context of ISC, but it's indeed a bit out of place here, so I removed it.